examples/ERC721TokenReceiver/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: build clean
+
+build:
+	test -d "openzeppelin-contracts-4.7.3" || tar -xf openzeppelin-contracts-4.7.3.tar.gz
+	mkdir -p build
+	solc --pretty-json --combined-json abi,bin oz_erc721.sol > build/oz_erc721.json
+	solc --pretty-json --combined-json abi,bin oz_erc721_tokenreceiver.sol > build/oz_erc721_tokenreceiver.json
```

Comment:
You should not rely on solc in the build environment and you should not commit giant tar balls like this into the repo. You can customize the Dockerfile for the build, like this --- https://github.com/reach-sh/reach-lang/blob/master/examples/remote/Dockerfile

Reply:
well, in my defense, we have two other copies of the entire openzeppelin repo sitting around :p

Reply:
No, we don't! They are downloaded as needed, not committed, just like I'm suggesting